packages/vscode-extension/src/builders/buildAndroid.ts

```diff
@@ -76,14 +76,38 @@ export async function buildAndroid(
   outputChannel: OutputChannel,
   progressListener: (newProgress: number) => void
 ): Promise<AndroidBuildResult> {
+  const { buildScript, env, android } = getLaunchConfiguration();
+
+  if (buildScript) {
+    const buildProcess = cancelToken.adapt(
+      exec(buildScript.name, ["android", ...(buildScript.args ?? [])])
+    );
+    let apkPath: string | undefined;
+    lineReader(buildProcess).onLineRead((line) => {
+      apkPath = line.trim();
```

Comment:
is it always just one line? Is is the output EAS returns?

Reply:
We don't want to assume that it's always EAS. We expect a script that outputs a single line in stdout. I forgot to do that but we should update docs to document this new option.

Reply:
I think it'd be easier if we let the script output stuff to stdout, otherwise people may need to add stdout forwarding to the scripts they already use and that'd make the scripts less readable. Instead, we can expect that when successful, the script returns code 0 and the path is the last line from the output.

Given that, the original code is ok, as it remembers the last line that was printent. However, we can make it so that the rest of the output is forwarded to extension logs such that it may be possible to get some feedback from the build script or see it output much like we see the output of gradle runs.